pipe_segment/transform/create_segment_map.py

```diff
@@ -1,3 +1,5 @@
+import logging
```

Comment:
Seems like you added loggingg at some point on this file and then removed it? If we are not logging anything I'd prefer to avoid the 'unused variable' warning from pyright.

Reply:
The logging was already there. This is not a PEP8 error, otherwise the CI checks would have failed. I don't know why pyright gives you a warning. It is pretty common to have logging configured in every module. What I have in my editor is a linter to check PEP8 errors. We can remove it if it's really bothering you though.

Reply:
I removed the unused logger.